Bottom: 36fa160c418f4d582c245c45fe717b30d8b31d14
Top:    d36befb7267a1451a20de678125ab6f88dc80dd8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-28 22:58:43 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
